editor/editor_help.h

```diff
@@ -325,20 +325,22 @@ class EditorHelpBit : public VBoxContainer {
 class EditorHelpBitTooltip : public PopupPanel {
 	GDCLASS(EditorHelpBitTooltip, PopupPanel);
 
+	static bool _is_tooltip_visible;
 	Timer *timer = nullptr;
 	int _pushing_input = 0;
 	bool _need_free = false;
 
 	void _start_timer();
 	void _safe_queue_free();
 	void _target_gui_input(const Ref<InputEvent> &p_event);
+	static Control *_make_invisible_control();
```

Comment:
I don't know if it's a best practice, but I personally prefer to place static methods before non-static ones.